solr/core/src/java/org/apache/solr/handler/component/QueryElevationComponent.java

```diff
@@ -1221,7 +1217,7 @@ public Elevation(Set<BytesRef> elevatedIds, Set<BytesRef> excludedIds, String qu
         includeQuery = EMPTY_QUERY;
         this.elevatedIds = Collections.emptySet();
       } else {
-        this.elevatedIds = ImmutableSet.copyOf(elevatedIds);
+        this.elevatedIds = Set.copyOf(elevatedIds);
```

Comment:
The issue is here :( `ImmutableSet.copyOf` keeps things in order it looks like. `Set.copyOf` doesn't care.